monitoring/src/deposit-monitor.ts

```diff
@@ -1,39 +1,79 @@
 import { BigNumber } from "ethers"
 
 import { SystemEventType } from "./system-event"
-import { context } from "./context"
+import { context, Environment } from "./context"
 
 import type { SystemEvent, Monitor as SystemEventMonitor } from "./system-event"
 import type { DepositRevealedEvent as DepositRevealedChainEvent } from "@keep-network/tbtc-v2.ts/dist/src/deposit"
 import type { Bridge } from "@keep-network/tbtc-v2.ts/dist/src/chain"
 
+const satsToRoundedBTC = (sats: BigNumber): string =>
+  (sats.div(BigNumber.from(1e6)).toNumber() / 100).toFixed(2)
+
+const hashUrls = (chainEvent: DepositRevealedChainEvent) => {
+  let fundingHashUrlPrefix = ""
+  if (context.environment === Environment.Mainnet) {
+    fundingHashUrlPrefix = "https://mempool.space/tx/"
+  }
+  if (context.environment === Environment.Testnet) {
+    fundingHashUrlPrefix = "https://mempool.space/testnet/tx/"
+  }
+
+  let revealHashUrlPrefix = ""
+  if (context.environment === Environment.Mainnet) {
+    revealHashUrlPrefix = "https://etherscan.io/tx/"
+  }
+  if (context.environment === Environment.Testnet) {
+    revealHashUrlPrefix = "https://goerli.etherscan.io/tx/"
+  }
```

Comment:
I would simplify this part a bit and use a `switch` statement that would make it cleaner:
```suggestion
  let fundingHashUrlPrefix = ""
  let revealHashUrlPrefix = ""
  
  switch(context.environment) { 
   case Environment.Mainnet: { 
      fundingHashUrlPrefix = "https://mempool.space/tx/"
      revealHashUrlPrefix = "https://etherscan.io/tx/"
      break
   } 
   case Environment.Testnet: { 
      fundingHashUrlPrefix = "https://mempool.space/testnet/tx/"
      revealHashUrlPrefix = "https://goerli.etherscan.io/tx/"
      break
   }
} 
```

Reply:
[02d32a](https://github.com/keep-network/tbtc-v2/pull/612/commits/02d32a8f4572403b146110c23e1c579fc3ca33a5)